src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -89,6 +91,7 @@ public VendorListService(String cacheDir,
         this.cacheDir = Objects.requireNonNull(cacheDir);
         this.endpointTemplate = Objects.requireNonNull(endpointTemplate);
         this.defaultTimeoutMs = defaultTimeoutMs;
+        this.deprecated = deprecated;
```

Comment:
(!) Seems we have a dependencies between `deprecated` and `fallbackVendorList` fields.
It is reasonable for case when `deprecated == true` to check `fallbackVendorList` is not NULL and throw exception if not so.

In other words, this is configuration misleading and we need to fail application start up if was no fallback GVL determined for deprecated GVL version.